src/constants/routes.js

```diff
@@ -0,0 +1,8 @@
+export const ROUTES = {
+  todo: 'todo',
+  todoTitle: 'Todo',
+  book: 'book',
+  bookTitle: 'Books',
+  navbar: 'navbar',
+  applicationTitle: 'Training APP'
+};
```

Comment:
This file should only have route names. Titles should go in another file. Also, use CamelCase for the routes